kubernetes/che-plugin-registry/templates/ingress.yaml

```diff
@@ -0,0 +1,24 @@
+#
+# Copyright (c) 2012-2017 Red Hat, Inc.
```

Comment:
Date should be 2018 (btw in 2012 it was not also started)

Reply:
fixed